docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
+                    "What is Spark?",
+                ]
+            }
+
+      - For this input format, MLflow will construct the appropriate request payload to the model endpoint type. For example, if your model is a chat endpoint (``llm/v1/chat``), MLflow will wrap your input string with the chat messages format like ``{"messages": [{"role": "user", "content": "What is MLflow?"}]}``. If you want to customize the request payload e.g. including system prompt, please use the next format.
+
+    * - A pandas DataFrame with a dictionary column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    {
+                        "messages": [
+                            {"role": "system", "content": "Please answer."},
+                            {"role": "user", "content": "What is MLflow?"},
+                        ],
+                        "max_tokens": 100,
+                    },
+                    # ... more dictionary records
+                ]
+            }
+
+      - In this format, the dictionary should have the correct request format for your model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                ["What is MLflow?"],
+                ["What is Spark?"],
+            ]
+
+      - The :py:func:`mlflow.evaluate()` will also accepts a list input. One notable requirement is that the each
+        list element i.e. input string needs to be wrapped in another list, so they can be passed as a single prediction request to the model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                [
+                    {
+                        "messages": [
+                            {"role": "system", "content": "Please answer."},
+                            {"role": "user", "content": "What is MLflow?"},
+                        ],
+                        "max_tokens": 100,
+                    },
+                    # ... more dictionary records
+                ]
+            ]
+
+      - Similar requirements as the above list format apply here as well.
+
+
+
+Passing Inference Parameters
+****************************
+
+You can pass additional inference parameters such as ``max_tokens``, ``temperature``, ``n``, to the model endpoint by setting the ``inference_params`` argument in :py:func:`mlflow.evaluate()`. The ``inference_params`` argument is a dictionary that contains the parameters to be passed to the model endpoint. The specified parameters are used for all the input record in the evaluation dataset.
+
+.. note::
+
+    When your input is a dictionary format tha represents request payload, it can also include the parameters like ``max_tokens``. If there are overlapping parameters in both the ``inference_params`` and the input data, the values in the ``inference_params`` will take precedence.
```

Comment:
```suggestion
    When your input is a dictionary format that represents request payload, it can also include the parameters like ``max_tokens``. If there are overlapping parameters in both the ``inference_params`` and the input data, the values in the ``inference_params`` will take precedence.
```